Tasks/Trots/Task2/Task2.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+
+from datetime import datetime
+
+d = {0: ("Двенадцать часов", "минут", "Первого"),
+     1: ("Один час", "Одна минута", "Второго", "Одной минуты"),
+     2: ("Два часа", "Две минуты", "Третьего", "Двух минут", "Двадцать"),
+     3: ("Три часа", "Три минуты", "Четвертого", "Трех минут", "Тридцать"),
+     4: ("Четыре часа", "Четыре минуты", "Пятого", "Четырех минут", "Сорок"),
+     5: ("Пять часов", "Пять минут", "Шестого", "Пяти минут"),
+     6: ("Шесть часов", "Шесть минут", "Седьмого", "Шести минут"),
+     7: ("Семь часов", "Семь минут", "Восьмого", "Семи минут"),
+     8: ("Восемь часов", "Восемь минут", "Девятого", "Восьми минут"),
+     9: ("Девять часов", "Девять минут", "Десятого", "Девяти минут"),
+     10: ("Десять часов", "Десять минут", "Одинадцатого", "Десяти минут"),
+     11: ("Одинадцать часов", "Одинадцать минут", "Двенадцатого", "Одинадцати минут"),
+     12: ("Двенадцать часов", "Двенадцать минут", "Первого", "Двенадцати минут"),
+     13: ("час", "Тринадцать минут", "", "Тринадцати минут"),
+     14: ("", "Четырнадцать минут", "", "Четырнадцати минут"),
+     15: ("", "Пятнадцать минут", "", "Пятнадцати минут"),
+     16: ("", "Шестнадцать минут"),
+     17: ("", "Семнадцать минут"),
+     18: ("", "Восемнадцать минут"),
+     19: ("", "Девятнадцать минут")}
```

Comment:
it's nice to know that you see a logical pattern here, but I would suggest to not mix your data in a one single dictionary because the structure became to fluid